aiida/cmdline/commands/cmd_database.py

```diff
@@ -35,14 +35,67 @@ def database_migrate(force):
     profile = manager.get_profile()
     backend = manager._load_backend(schema_check=False)  # pylint: disable=protected-access
 
-    if not force:
-        echo.echo_warning('Migrating your database might take a while.')
-        echo.echo_warning('Before continuing, make sure the daemon is stopped and you have a backup of your database.')
-        echo.echo_warning('', nl=False)
-        confirm_message = 'Are you really sure you want to migrate the database for profile "{}"?'.format(profile.name)
-        click.confirm(confirm_message, abort=True)
+    if force:
+        raise NotImplementedError('--force is disabled currently to prevent accidental migrations.')
+
+    echo.echo('\n' + '*' * 79 + '\n')
+    echo.echo_warning('Before continuing, make sure the daemon is stopped and you have a backup of your database.')
+    echo.echo_warning(
+        'It is ESSENTIAL to have a backup of your database - once you do this migration you can NEVER go back!')
+    echo.echo_warning('You will be committed to this new version of AiiDA.')
+    echo.echo_warning('Please be patient - migrating your database might take a while.')
 
-    backend.migrate()
+    echo.echo('\n' + '*' * 79 + '\n')
+
+    try:
+        # First prompt
+        backup_prompt = 'Have you got a back up of the database for profile "{}"?'.format(profile.name)
+        echo.echo_warning(backup_prompt)
+        echo.echo_warning('', nl=False)
+        confirm_message = 'Type "I DO HAVE A BACKUP"'
+        response = ''
+        while response != 'I DO HAVE A BACKUP':
+            response = click.prompt(confirm_message)
+
+        # Second prompt
+        echo.echo('')
+        echo.echo_warning('Have you stopped the daemon?')
+        echo.echo_warning('', nl=False)
+        confirm_message = 'Type "I HAVE STOPPED THE DAEMON"'
+        response = ''
+        while response != 'I HAVE STOPPED THE DAEMON':
+            response = click.prompt(confirm_message)
+
+        # Final prompt
+        echo.echo('')
+        message = 'Are you absolutely ready to migrate profile "{}"? This will be PERMENANT.'.format(profile.name)
```

Comment:
ahah I like this :-D only typo PERMANANT -> PERMANENT, apart for this it can be approved